cmd/test/main.go

```diff
@@ -281,15 +284,15 @@ func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error
 		}
 	}
 
-	var invocationFailedErr error = nil
-
 	if err != nil {
-		invocationFailedErr = err
-	} else if validationManager.InvalidYamlFilesCount() > 0 || validationManager.InvalidK8sFilesCount() > 0 || results.EvaluationResults.Summary.TotalFailedRules > 0 {
-		invocationFailedErr = fmt.Errorf("")
+		return err
 	}
 
-	return invocationFailedErr
+	if validationManager.InvalidYamlFilesCount() > 0 || validationManager.InvalidK8sFilesCount() > 0 || results.EvaluationResults.Summary.TotalFailedRules > 0 {
```

Comment:
extract this to function with a clear name of what it does, it looks like there is a meaning here as to why you return the violations error but I have to read the whole thing to get it